public/usage-examples/graphics/fill_rectangle_on_bitmap/fill_rectangle_on_bitmap-1-simple-top-level.cs

```diff
@@ -0,0 +1,52 @@
+﻿using SplashKitSDK;
+using static SplashKitSDK.SplashKit;
+
+//Create a new window with title and dimensions 

```

Comment:
Check that you comments have a space after `//`  it is currently inconsistent across files